install/cameras.csv

```diff
@@ -24,7 +24,10 @@ Mobotix AG,M15,10.16.105.12,N,0,87,4,0,0,2019-05-23T00:00:00Z,9999-01-01T00:00:0
 Mobotix AG,M15,10.17.107.14,MTSR,0,0,2,0,0,2016-02-02T02:00:00Z,9999-01-01T00:00:00Z,Mangateitei looking at South Ruapehu
 Mobotix AG,M15,10.18.101.137,CVTP,10,290,-2.5,0,0,2016-07-11T00:03:00Z,2019-06-02T23:40:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
 Mobotix AG,M15,10.18.101.137,CVTP,-8,290,-2.5,0,0,2019-06-02T23:50:00Z,2019-06-19T22:00:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe: knocked off-alignment by ice
-Mobotix AG,M15,10.18.101.137,CVTP,10,290,-2.5,0,0,2019-06-19T22:00:00Z,9999-01-01T00:00:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,10,290,-2.5,0,0,2019-06-19T22:00:00Z,2019-07-06T21:10:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,4,290,-2.5,0,0,2019-07-06T21:20:00Z,2019-07-08T00:40:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,0.5,290,-2.5,0,0,2019-07-08T00:50:00Z,2019-11-27T20:50:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,9.5,290,-2.5,0,0,2019-11-27T21:00:00,9999-01-01T00:00:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
```

Comment:
You need to append a "Z" to the start time on the last changed row, i.e. `2019-11-27T21:00:00Z`.